<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgQueuesNewPatchDialog</class>
 <widget class="QDialog" name="HgQueuesNewPatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Patch</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="toolTip">
      <string>Enter the patch name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Message:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="messageEdit">
     <property name="toolTip">
      <string>Enter the commit message for the patch</string>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="userGroup">
     <property name="toolTip">
      <string>Select to give user information</string>
     </property>
     <property name="title">
      <string>User</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="currentUserCheckBox">
        <property name="toolTip">
         <string>Select to use the name of the current user</string>
        </property>
        <property name="text">
         <string>Use current user</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userEdit">
        <property name="toolTip">
         <string>Enter the user name to be used for the patch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="dateGroup">
     <property name="toolTip">
      <string>Select to give date and time information</string>
     </property>
     <property name="title">
      <string>Date and Time</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="currentDateCheckBox">
        <property name="toolTip">
         <string>Select to use the current date and time</string>
        </property>
        <property name="text">
         <string>Use current date and time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date/Time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="toolTip">
         <string>Enter the date and time to be used for the patch</string>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd hh:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>241</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>messageEdit</tabstop>
  <tabstop>userGroup</tabstop>
  <tabstop>currentUserCheckBox</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>dateGroup</tabstop>
  <tabstop>currentDateCheckBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgQueuesNewPatchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgQueuesNewPatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentUserCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentUserCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentDateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentDateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
